gps/app/views/ChooseProvider.js

```diff
@@ -408,11 +409,23 @@ const styles = StyleSheet.create({
     flexDirection: 'column',
     textAlignVertical: 'top',
     justifyContent: 'flex-start',
-    padding: 10,
+    padding: 20,
     width: '96%',
     alignSelf: 'center',
     backgroundColor: Colors.WHITE,
   },
+  startLoggingButtonTouchable: {
```

Comment:
If you can make the check name "eslint" and not "ESlint check" it will show under the right place in actions: 

At the moment it shows in i18n: https://github.com/Path-Check/covid-safe-paths/pull/872/checks?check_run_id=691207449

Reply:
Are there two eslint checks? I see that one and also the "lint" action

Reply:
To clarify, There is only one eslint check, but the new action we used sets the annotations source to a different job name and so it doesnt match up..  so it looks like there are two checks.

The prior eslint checker maps the it's same job name (via with: config), so it lines up better.

Problem is it doesn't tell you in the job log what failed. ... grr.